src/main/java/com/example/TicketFinder/Concert.java

```diff
@@ -0,0 +1,19 @@
+package com.example.TicketFinder;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+class Concert {
+    private String nameOfConcert;
+    private LocalDate date;
```

Comment:
I'm not sure if LocalDate is appropriate here. LocalDate can store the only date, e.g. 29.02.2020 without the hours.